### DISCUSSION TITLE
Rename Spaces

### DISCUSSION THREAD

**Discussion by m1nkeh** [rxhanson/Rectangle#1408] [Ideas]
I don't know if this has come up before, but is there a way that Rectangle could also rename spaces?

It closely falls under window management I guess, so thought I'd ask! 😊

Keep up the good work.

**Comment by rxhanson** [comment]
Thanks for the feedback! I've investigated Spaces quite a bit while making [Space Capsule](https://spacecapsule.app). Unfortunately there is no public API available for renaming Spaces. In fact, there's not much of a public API at all for anything dealing with Spaces.

**Comment by m1nkeh** [comment]
Ok, thank you. That's a shame... I can't believe this isn't a feature in macOS, heh 😅
